src/main/controllers/helpers/DocumentHelpers.ts

```diff
@@ -37,32 +37,20 @@ export const getDocumentAdditionalInformation = async (doc: Document, accessToke
 export const combineDocuments = (...arrays: DocumentDetail[][]): DocumentDetail[] =>
   [].concat(...arrays.filter(Array.isArray)).filter(doc => doc !== undefined);
 
-export const createDownloadLink = (file: Document, multipleFiles?: boolean): string => {
+export const createDownloadLink = (file: Document): string => {
   const mimeType = getFileExtension(file?.document_filename);
   let downloadLink = '';
   if (file && file.document_size && file.document_mime_type && file.document_filename) {
-    if (multipleFiles) {
-      downloadLink =
-        "<a href='/getTribunalOrderDocument/:docId' target='_blank' class='govuk-link'>" +
-        file.document_filename +
-        '(' +
-        mimeType +
-        ', ' +
-        formatBytes(file.document_size) +
-        ')' +
-        '</a>';
-      downloadLink = downloadLink.replace(':docId', getDocId(file.document_url));
-    } else {
-      downloadLink =
-        "<a href='/getSupportingMaterial' target='_blank' class='govuk-link'>" +
-        file.document_filename +
-        '(' +
-        mimeType +
-        ', ' +
-        formatBytes(file.document_size) +
-        ')' +
-        '</a>';
-    }
+    downloadLink =
```

Comment:
Could this be rewritten using template literals?

Reply:
Added template literal